ResearchKit/Common/ORKEligibilitySelectionView.h

```diff
@@ -0,0 +1,77 @@
+/*
+ Copyright (c) 2015, Apple Inc. All rights reserved.
+ 
+ Redistribution and use in source and binary forms, with or without modification,
+ are permitted provided that the following conditions are met:
+ 
+ 1.  Redistributions of source code must retain the above copyright notice, this
+ list of conditions and the following disclaimer.
+ 
+ 2.  Redistributions in binary form must reproduce the above copyright notice,
+ this list of conditions and the following disclaimer in the documentation and/or
+ other materials provided with the distribution.
+ 
+ 3.  Neither the name of the copyright holder(s) nor the names of any contributors
+ may be used to endorse or promote products derived from this software without
+ specific prior written permission. No license is granted to the trademarks of
+ the copyright holders even if such marks are included in this software.
+ 
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+ FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+
+#import <Foundation/Foundation.h>
+#import <ResearchKit/ResearchKit.h>
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+static const CGFloat EligibilityButtonHeight = 130;
+
+/**
+ Buttons used for `YES` or `NO` values for elgibility.
+ */
+ORK_CLASS_AVAILABLE
+@interface ORKEligibilityButton : UIButton
+
++ (id)yesEligibilityButton;
+
++ (id)noEligibilityButton;
+
+@end
+
+
+@class ORKEligibilitySelectionView;
+
+@protocol ORKEligibilitySelectionViewDelegate <NSObject>
+
+- (void)selectionViewSelectionDidChange:(ORKEligibilitySelectionView *)view;
+
+@end
+
+
+/**
+ The `ORKEligibilitySelectionView` class represents the boolean buttons of 
+ the eligibilty questions.
+ */
+ORK_CLASS_AVAILABLE
+@interface ORKEligibilitySelectionView : UIView
```

Comment:
I prefer selection view because it functions similarly to the `ORKImageSelectionView`. When an option is selected, it deselects any other ones. And it also has the option to unselect completely, which defies the nature of binary, since we have options: yes, no, and nil.